packages/api/src/routes/settings.ts

```diff
@@ -57,13 +97,14 @@ const webhookURLExactBlacklist = ["0"];
 
 class SettingsDTO {
   public constructor(
-    public id: string,
+    public key: string,
```

Comment:
why is this now `key`? doesn't this refer to the PG id?

Reply:
Good catch this was an accident